docker/docker_image_src.go

```diff
@@ -83,6 +83,26 @@ func (s *dockerImageSource) GetManifest() ([]byte, string, error) {
 	return s.cachedManifest, s.cachedManifestMIMEType, nil
 }
 
+// GetTargetManifest ...
+func (s *dockerImageSource) GetTargetManifest(reference string) ([]byte, string, error) {
```

Comment:
The method declaration quoted above, and “new method in the image source interface”, are in `ImageSource`.

(To be clear, `ImageSource.GetManifestByDigest` is perfectly fine; I didn’t like the rch/platform logic in the sources.)